src/base/bittorrent/trackerentry.cpp

```diff
@@ -89,24 +107,42 @@ void TrackerEntry::setTier(const int value)
 int TrackerEntry::numSeeds() const
 {
     int value = -1;
-    for (const lt::announce_endpoint &endpoint : nativeEntry().endpoints)
+    for (const lt::announce_endpoint &endpoint : nativeEntry().endpoints) {
+#if (LIBTORRENT_VERSION_NUM >= 20000)
+        for (const lt::announce_infohash &infoHash : endpoint.info_hashes)
```

Comment:
What does "announce info hash" means?

Reply:
See this: https://github.com/arvidn/libtorrent/blob/RC_2_0/include/libtorrent/announce_entry.hpp#L156